packages/bpk-stylesheets/base.css

```diff
@@ -240,8 +240,7 @@ figure {
  * 2. Show the overflow in Edge and IE.
  */
 hr {
-  -webkit-box-sizing: content-box;
-          box-sizing: content-box;
+  box-sizing: content-box;
```

Comment:
Will this cause browser support issues by this changing?

Reply:
I feel like my comment addresses that already 😅

Reply:
Wait sorry, I see you marked this as resolved. Never mind!

Reply:
Yeah I think I clicked submit as you did so our comments went in at the same time